### ISSUE TITLE
Enable search/replace in text editors

### ISSUE THREAD

**Issue by ulitol97** [weso/rdfshape-client#46]
Reference/proof of concept: https://codemirror.net/demo/search.html

**Comment by mistermboy** [comment]
Both YASHE and Yate now support search and replace. (https://github.com/perfectkb/yate/pull/2)
However, I have been looking at rdfshape-client and it seems that Yate is not in use, am I wrong? @labra @ulitol97

**Comment by ulitol97** [comment]
Awesome!
About the usage of Yate, I don't know if @labra has anything to add but afaik you are right. I'm modifying the codebase to use Yate whenever the input data format is turtle.

**Comment by labra** [comment]
I think I switched off Yate because it was giving me some problems and I wanted the syntax highlihgting functionality to be quickly available. 

If you are able to make it work again, it would be great.

**Comment by ulitol97** [comment]
I'm my last PR I've made changes so that we use Yate again. I've also updated Yate to its latest version in order to implement the search/replace functionality.

I've noticed a couple of issues:
- Search/replace is available in Yashe but not in Yate.
- Replace is not working on Yashe (an error in shown in console whenver Ctrl + Shift + F is used)

@mistermboy any help/clues? I'm probably missing something.

[Here](http://rdfshape.weso.es/link/16088093471) is an example of a ShEx validation using both Yate and Yashe.
